its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -766,6 +769,32 @@ public void testDuplicateAnalyzersWithSameNameAreNotRemoved() throws IOException
     assertThat(TestUtils.getMeasureAsInteger("DuplicateAnalyzerReferences", "files", ORCHESTRATOR)).isEqualTo(2);
   }
 
+  @Test
+  public void testIgnoreIssuesDoesNotRemoveSourceGenerator() throws IOException {
+    assumeFalse(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2017")); // We can't run .NET Core SDK under VS 2017 CI context
+    Path projectDir = TestUtils.projectDir(temp, "ProjectWithSourceGenerator");
+
+    String token = TestUtils.getNewToken(ORCHESTRATOR);
+    String folderName = projectDir.getFileName().toString();
+
+    ScannerForMSBuild scanner = TestUtils.newScannerBegin(ORCHESTRATOR, "ProjectWithSourceGenerator", projectDir, token, ScannerClassifier.NET_FRAMEWORK_46)
+      // exclude test projects
+      .setProperty("sonar.cs.roslyn.ignoreIssues", "true")
+      .setProperty("sonar.verbose", "true");
+
+    ORCHESTRATOR.executeBuild(scanner);
+
+    TestUtils.runMSBuild(ORCHESTRATOR, projectDir,"/t:Rebuild");
+
+    BuildResult result = TestUtils.executeEndStepAndDumpResults(ORCHESTRATOR, projectDir, "ProjectWithSourceGenerator", token);
```

Comment:
can you check the build result logs to make sure they contain the SA warnings? (to ensure the analyzers are not removed?)

Reply:
Ok, #1211 indeed is about code generators.